src/neurostatslib/utils.py

```diff
@@ -109,3 +109,45 @@ def plot_spike_raster(
     )
     ax.set(yticks=[], xlim=[0, spike_data.shape[1]])
     return ax
+
+
+def row_wise_kron(A: jnp.array, C: jnp.array, jit=False, transpose=True) -> jnp.array:
+    """
+    Compute the row-wise Kronecker product between two matrices using JAX.
```

Comment:
can we link to what the Kronecker product means?

Reply:
added a link to wikipedia, if you have better online research feel free to change